notebooks/tests/__init__.py

```diff
@@ -61,24 +62,29 @@ def _get_unzip(filename: str, url: str) -> None:
 
 
 def fetch_spark() -> None:
+    """Download and unzip Spark."""
     _get_unzip(_SPARK_FILENAME, _SPARK_URL)
-    os.environ['SPARK_HOME'] = os.path.join(os.getcwd(), _SPARK_FILENAME)
+    os.environ["SPARK_HOME"] = os.path.join(os.getcwd(), _SPARK_FILENAME)
 
 
 def _get_hadoop() -> None:
     _get_unzip(_HADOOP_FILENAME, _HADOOP_URL)
-    os.environ['HADOOP_HOME'] = os.path.join(os.getcwd(), _HADOOP_FILENAME)
+    os.environ["HADOOP_HOME"] = os.path.join(os.getcwd(), _HADOOP_FILENAME)
 
 
 def _copy_all_hadoop_jars_to_pyflink() -> None:
     _get_hadoop()
     if not os.getenv("HADOOP_HOME"):
-        raise Exception("The HADOOP_HOME env var must be set and point to a valid Hadoop installation")
+        raise Exception(
+            "The HADOOP_HOME env var must be set and point to a valid Hadoop installation"
+        )
 
     pyflink_lib_dir = _find_pyflink_lib_dir()
-    for i, jar in enumerate(_jar_files()):
+    for _i, jar in enumerate(_jar_files()):
```

Comment:
why these changes?

Reply:
flake8 was failing for me locally, not sure why it wasn't picked up by CI